client/lib/wpcom-xhr-wrapper/index.js

```diff
@@ -20,7 +15,7 @@ export default async function ( params, callback ) {
 	return xhr( params, function ( error, response, headers ) {
 		if ( error && error.name === 'InvalidTokenError' ) {
 			debug( 'Invalid token error detected, authorisation probably revoked - logging out' );
-			userUtils.logout();
+			window.location.reload();
```

Comment:
@cbauerman Is there a reason why you might want to revert this change? Did it break anything in Jetpack Cloud? I noticed that you created a `revert/wpcom-xhr-bad-token-fallback` branch for that 🙂

Reply:
Yeah, I was wondering the same thing, which is why I asked here: p1624086030240200/1624047611.234600-slack-CQXNTE9K9 . I'm keeping an eye on this and will revisit once we have more information.